cmd/opm/validate/validate.go

```diff
@@ -0,0 +1,50 @@
+package validate
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/pkg/lib/config"
+)
+
+func NewConfigValidateCmd() *cobra.Command {
+	validate := &cobra.Command{
+		Use:   "validate <directory>",
+		Short: "Validate the declarative index config",
+		Long:  "Validate the declarative config JSON file(s) in a given directory",
+		Args: func(cmd *cobra.Command, args []string) error {
+			return cobra.ExactArgs(1)(cmd, args)
+		},
+		RunE: configValidate,
+	}
+
+	validate.Flags().BoolP("debug", "d", false, "enable debug log output")
+	return validate
+}
+
+func configValidate(cmd *cobra.Command, args []string) error {
+	debug, err := cmd.Flags().GetBool("debug")
+	if err != nil {
+		return err
+	}
```

Comment:
I realize we're doing almost no logging for this run command, so if we don't run into a stat error, then we're essentially running `opm validate <directory>` and no output is being returned.

Reply:
Correct. There is no logger hookup on any of validation funcs that we are using here. Frankly, there isn't much to log because there is one-time atomic run kind of a command. There ain't a lot of operations to log. Most of the time, something happens meaning there are errors and should return back.